packages/atlas/lib/src/polyline.dart

```diff
@@ -13,10 +13,13 @@ class Polyline {
   /// The `Color` of the line
   final Color color;
 
+  final int width;
```

Comment:
Please add documentation describing this new property. Thanks

Reply:
ok i add documentation